Author: Jonas <spameier+alpine@proton.me>
Summary: Replaces scripts with entry points (part of upstream commit https://github.com/dirkjanm/ldapdomaindump/commit/d559463eb29857f2660bf3867bfb9f8610d1ddb1)
diff --git a/bin/ldapdomaindump b/bin/ldapdomaindump
deleted file mode 100644
index 3b76994..0000000
--- a/bin/ldapdomaindump
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/usr/bin/env python
-import ldapdomaindump
-ldapdomaindump.main()
\ No newline at end of file
diff --git a/bin/ldd2bloodhound b/bin/ldd2bloodhound
deleted file mode 100644
index 4340802..0000000
--- a/bin/ldd2bloodhound
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/usr/bin/env python
-from ldapdomaindump import convert
-convert.ldd2bloodhound()
diff --git a/bin/ldd2pretty b/bin/ldd2pretty
deleted file mode 100644
index 82a9136..0000000
--- a/bin/ldd2pretty
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/usr/bin/env python
-from ldapdomaindump import pretty
-pretty.main()
diff --git a/setup.py b/setup.py
index aaf02f0..213f669 100644
--- a/setup.py
+++ b/setup.py
@@ -10,6 +10,8 @@ setup(name='ldapdomaindump',
       install_requires=['dnspython', 'ldap3>=2.5,!=2.5.2,!=2.5.0,!=2.6'],
       package_data={'ldapdomaindump': ['style.css']},
       include_package_data=True,
-      scripts=['bin/ldapdomaindump', 'bin/ldd2bloodhound', 'bin/ldd2pretty'],
+      entry_points= {
+        'console_scripts': ['ldapdomaindump=ldapdomaindump:main','ldd2bloodhound=ldapdomaindump.convert:ldd2bloodhound','ldd2pretty=ldapdomaindump.pretty:main']
+      },
       license="MIT",
       )
